src/interfaces/api/v2/topic.interface.ts

```diff
@@ -31,7 +31,16 @@ export interface ITopic extends IApiResource {
   related_from: number[];
 }
 
+export interface ITopicLocalName {
```

Comment:
See my other comment about local vs remote name. It often causes a lot of confusion and is only used in particular circumstances. We can circle back on the discussion for this.